ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,31 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+namespace StrongPasswordCheck {
+   class Program {
+      static void Main (string[] args) {
+         Console.WriteLine ("Enter the password:");
+         string password = Console.ReadLine ();
+         if (isStrongPassword (password))
+            Console.WriteLine ("The password is strong.");
+         else Console.WriteLine ("The password is not strong.Please ensure that the pasword contains atleast six characters-atleast one digit,atleast one upper and lowercase letter and atleast one special character !@#$%^&()-+");

```

Comment:
Try to print the exact reason why this is not a strong password. for eg welcome@1 -> not strong because it does not have an uppercase letter